### PR TITLE
Better message when no agreement found.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: cmd/listagreements/listagreements.go [hunk 1]
```diff
@@ -82,6 +82,9 @@ func (c *listAgreementsCommand) Run(ctx *cmd.Context) error {
 
 	agreements, err := apiClient.GetUsersAgreements()
 	if err != nil {
+		if errors.IsNotFound(errors.Cause(err)) {
+			return errors.Errorf("no agreement found")
```
Comment (mattyw) [inline cmd/listagreements/listagreements.go]: I'd say don't return an error - return that we know exactly what's happening, but that the user has no agreements

**Comment by tasdomas** [comment]
Should we add a test?

**Comment by tasdomas** [comment]
LGTM

**Comment by mattyw** [comment]
We could change this server side

**Comment by cmars** [comment]
Agree with @mattyw. A 404 doesn't seem right to me: it's not that the list doesn't exist -- it's just empty.

**Comment by alesstimec** [comment]
@cmars @mattyw istm that we always return 404 when a resource doesn't exist..

**Comment by cmars** [comment]
We could say that the resource in this case is the list of agreements itself, which exists, if it's a valid user, but is empty, if there are no agreements made yet.

**Pull request closed without merging**